package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -16,6 +16,7 @@ CONFIGFILE="/var/etc/dnsmasq.conf"
 HOSTFILE="/tmp/hosts/dhcp"
 TRUSTANCHORSFILE="/usr/share/dnsmasq/trust-anchors.conf"
 TIMESTAMPFILE="/etc/dnsmasq.time"
+TIMEVALIDFILE=/tmp/ntp.d/dnsmasqsec
```

Comment:
I've just sent a patch on the ML to fix the /var/state is world writable,
can you put TIMEVALIDFILE where it was (/var/state/dnsmasqsec)